rll/collectd.sh

```diff
@@ -259,11 +263,22 @@ else
   echo "swapfile not setup, skipping collectd swap plugin"
 fi
 
+# Populate RS_RLL_PORT
+source /var/run/rightlink/secret
```

Comment:
Its owned by rightlink and the script is run as rightlink, so works in this case